### PR TITLE
New package: termshark-1.0.0

### PR DESCRIPTION
**Pull request by Piraty**

### CODE REVIEW

**Comment by Piraty** [comment]
only one minor issue:
```
$ termshark -v
termshark v<localbuild>
```

**Comment by Anachron** [comment]
Use vsed to fix https://github.com/gcla/termshark/blob/b0f047255ce8c9023dd4e9aa16276deaa79fce89/version.go#L7 then?

**Comment by Piraty** [comment]
done.

File: srcpkgs/termshark/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'termshark'
+pkgname=termshark
+version=1.0.0
+revision=1
+build_style=go
+go_import_path="github.com/gcla/termshark"
+go_package="${go_import_path}/cmd/termshark"
+hostmakedepends="git"
+# termshark requires tshark
```
Comment (Duncaen) [inline srcpkgs/termshark/template]: what?

**Comment by Duncaen** [comment]
No use something like `go_ldflags="-X github.com/gcla/termshark.Version=v${version}"`.

**Pull request merged**